### PR TITLE
Added aria-label

### PR DESCRIPTION
**Pull request by CBID2**
I made line 77 more accessible

## Fixes Issue
[2382](https://github.com/EddieHubCommunity/LinkFree/issues/2382)

## Check List (Check all the applicable boxes) 

- [X ] My code follows the code style of this project.
- [ X] My change requires changes to the documentation.
- [X ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [ X] This PR does not contain plagiarized content.
- [X ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2506"></a>

### CODE REVIEW

File: components/navbar/Navbar.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ export default function Navbar() {
                         style: { verticalAlign: "middle" },
                       }}
                     >
-                      <FaGithub />
+                      <FaGithub aria-label=“GitHub”/>
```
Comment (dan-mba) [inline components/navbar/Navbar.js:77]: Change the quotes to the standard double quote. The stylized double quotes that iOS uses when I typed in Discord are causing React to fail.

**Review by dan-mba — APPROVED** [review]
Looks good to me.

**Review by Mahesh-Kasabe — APPROVED** [review]

**Review by CBID2 — COMMENTED** [review]
Great

**Comment by CBID2** [comment]
> Looks good to me.

Thanks @dan-mba! :) Now onto merging it

**Pull request merged**